src/test/ui/borrowck/move-in-pattern-mut-in-loop.stderr

```diff
@@ -0,0 +1,15 @@
+error[E0382]: use of moved value
+  --> $DIR/move-in-pattern-mut-in-loop.rs:7:21
+   |
+LL |         if let Some(mut _x) = opt {}
+   |                     ^^^^^^ value moved here, in previous iteration of loop
+   |
+   = note: move occurs because value has type `&mut i32`, which does not implement the `Copy` trait
+help: borrow this field in the pattern to avoid moving `opt.0`
```

Comment:
If it's `Some(x)`, we don't suggest `ref mut x` since I think the user should encounter a mutability error anyway (and it may be verbose). But if we should also suggest it, I'll do so (we could also tweak the wording in that case).

Reply:
I think it's probably fine too, because the user still gets a valid suggestion.